preprocess.py

```diff
@@ -0,0 +1,7 @@
+from pycparser import preprocess_file
+import re
+
+def preprocess(file, cpp_args=''):       
+    preprocessed = preprocess_file(file, cpp_args=cpp_args)
+    preprocessed = re.sub(r'#.*\n', '', preprocessed)
```

Comment:
instead of doing this, we could pass `-P` as cpp_args (`-nostdinc` also inhibits some line markers). With that in mind, does this deserve a whole new file and function? Do we have plans to grow it?

I guess if we start auto-injecting macros, e.g. if we start basing more stuff on AST traversals, then that would warrant a function... but we don't want such macros from where we call this function from test code.

Reply:
I'm new to python code structure, but at least with other other languages I go with the philosophy that if it can be though of as a object, it can be put in its own file. (Information hiding and reuse benefits).
I'll make the `-P` change

Reply:
Agreed in general, but yeah, it's a bit silly for a single one-line function that we don't expect to grow any further.